schemas/oic.sec.cred.json

```diff
@@ -62,7 +62,7 @@
           }
         }
       },
-      "required": [ "credid", "subjectuuid", "credtype" ]
```

Comment:
CR-1485 doesn't say anything about whether or not "credid" is mandatory or not.  I'm not sure which CR made the change but the latest version of the security spec that Habib sent out says that "credid" is mandatory in table 15.

Ned updated his CR to make "aceid" mandatory as well.